### DISCUSSION TITLE
Jet-Flo Vac Pump Test

### DISCUSSION THREAD

**Discussion by DeeEmm** [DIY-Flow-Bench/DIY-Flow-Bench#151] [General]
Decided to hook up the regenerative air pump I purchased a while back and do some testing to determine how much depression it could pull. This particular pump is unfortunately a little under-powered for measuring things like cylinder heads but should hopefully be fine for testing out the code with. 

This unit is the XGB-550 which will flow 62cfm. Somehow I screwed up when I ordered and ended up with a 500w pump instead of a 5000w pump. The pump I thought I ordered, the XGB-5500 flows 284cfm. A silly mistake but easily rectified by the application of more $$$'s (ouch).

For those who are interested here's the video of the test.

<img width="1281" alt="Screenshot 2023-05-12 at 9 24 03 pm" src="https://github.com/DeeEmm/DIY-Flow-Bench/assets/3038710/43618efb-d2b7-4627-864a-4b6d2f6d75b1">

https://youtu.be/MjXlDJHD5MM

Test results were about 6" WC with a 20mm saw blade orifice.

One thing I am impressed with is that these pumps are fairly quiet. I think that if you mounted this in some form of enclosure it would hardly be noticeable.  Heaps quieter than half a dozen vacuum motors under load.

Next steps are to get the MAF mounted up and the shield hooked up for some testing. 

I'm definitely considering buying the larger pump, despite the price tag. There are 3, 4 and 5.5 kw versions available locally but they are not exactly cheap.  The 5.5kw version will support just over 300cfm which is far more than anything I will ever need. I could probably get away with 200cfm but I think that at some point this will be too small. 

Will see how the testing goes before pulling the trigger.

**Comment by DeeEmm** [comment]
Hmmnnnn. Shopping for pumps last night I wonder if the rotation of the pump was incorrect. Most seem to have the discharge pipe on the side that I have the vac on. I suspect that the phase rotation is incorrect and the pump might be running backwards.

The impeller on these is not symmetrical so I suspect that if I correct this I may get a different result. I'll see what happens if I run up the VSD in reverse.

**Comment by DeeEmm** [comment]
OK. So I rewired the pump and re-ran the test with the pump running in the correct direction. The pump definitely performed better.

Test result was about 8" with the 20mm saw blade orifice

As a comparison I also hooked up the shop vac that I have. It is a 35Litre generic shop vac that I've seen sold under a number of different brands from outlets like Aldi and Supercheap Auto. It claims that it can flow 30 litres a second, which equates to about 63 cfm. What I found was that it seems to pull about the same as the vac pump. It might even be capable of a little more if I sealed the lid to the base.

Test result for shop vac was about 8" with the 20m orifice

It was however heaps noisier.

The takeaway from this is that the 20mm test orifice needs to be tested at somewhere around 6 inches of depression to be able to be used with a shop vac style setup. Also whilst the 500w pump is a nice solution, for the price the shop vac is a better (i.e. cheaper) option, unless you are happy to pay for the additional sound reduction. These pumps are definitely suitable and perform very closely to their stated specs for flow but a much larger pump than the 500w version is required for flowing cylinder heads.

The ideal pump should be the 5.5kW version, which should flow just over 300cfm according to the specs, however the relationship between flow and static pressure should not be overlooked as to achieve 28 inches of depression with a 50mm valve at full lift I suspect that an even larger pump may be required. The specs state that the pump can deliver 520m3/h and has a pressure rating of 48 kPa. but I'm not sure that it can do both at the same time as this is not what I saw with the smaller pump. 

48kPa is equivalent to 192 inches of water, which is massive, but I suspect that this is 192 inches at zero flow i.e. the delivery tube blocked. I have asked the vendor of they can provide a flow map.

Will keep you posted

Here's the spec sheet for the pump range for anyone who is interested

![s-l1600](https://github.com/DeeEmm/DIY-Flow-Bench/assets/3038710/54da7b79-fc5b-4f24-bc1f-497e71cbf4fd)

**Comment by DeeEmm** [comment]
Got a response from the local vendor...

![s-l1600](https://github.com/DeeEmm/DIY-Flow-Bench/assets/3038710/d556641c-8f57-414f-8e71-09975c6ef795)

Extrapolating the data from the graph above it looks like the 5.5kW pump can support approximately 240 cfm at a depression of 28"

**Comment by black-top** [comment]
My small pump 2BH1610-7hh36 is 4.3kw, and looks like it can be overdriven to 87Hz at that frequency it is 6.4kw. From the initial test it seems that the pressure may not be enough for high flow, to keep 28''h2o all way up. The big pump is about 10kw, but now I have no way to look up the exact model, but this should really be enough, it is several times bigger than the small one.

**Reply by DeeEmm** [reply]
I found that I could only overdrive mine to about 65hz. Above this the rotor started to slip. The torque drops off as the frequency increases. I assume that the load exceeded the torque.

I would think that the 10Kw pump would be ample.
